core/Cargo.toml

```diff
@@ -124,7 +124,7 @@ services-dbfs = []
 services-dropbox = []
 services-etcd = ["dep:etcd-client", "dep:bb8"]
 services-foundationdb = ["dep:foundationdb"]
-services-fs = ["tokio/fs"]
+services-fs = []
```

Comment:
services-fs does depend on `tokio/fs`:

https://github.com/apache/opendal/blob/755de5829a127e3e1f9da334e69cf4436578f889/core/src/services/fs/backend.rs#L233-L239

The build passed for other deps imples this.

Reply:
Good to know. Does `create_dir_all` need to be `async`? tokio fs does nothing better than move it to another thread and block. I wonder such an action won't block.